src/js/components/List/List.js

```diff
@@ -679,9 +681,9 @@ const List = React.forwardRef(
                     tag="li"
                     background={adjustedBackground}
                     border={adjustedBorder}
-                    isDisabled={isDisabled}
                     flex={false}
                     pad={pad || theme.list.item.pad}
+                    $isDisabled={isDisabled}
                     {...defaultItemProps}
                     {...boxProps}
                     {...clickProps}
```

Comment:
Do any of the props passed through here need to be transient?

Reply:
Great question, I will take a look.